973_K_Closest_Points_to_Origin.py

```diff
@@ -0,0 +1,22 @@
+# Time: O(n)
+# Space: O(n)
```

Comment:
The time complexity and space complexity are wrong.
Look into the time complexity of each method of heapq.